package.json

```diff
@@ -41,7 +41,8 @@
     "uglify"
   ],
   "engines": {
-    "node": "^14.13.1 || >=16.0.0"
+    "node": ">=16.14.0",
+    "npm": ">=8.0.0"
```

Comment:
This change makes not much sense.

The engine entry is only for consumers, not for contributors. So the original entry is correct. We currently support Node 14 and Node 16 as you can see via CI.

Regarding the npm version entry, this also makes not much sense because the lockfile uses lockfile version 3, which is only supported by npm 9 and newer. See also https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#lockfileversion

Reply:
We could also add an `.npmrc` file:

```
# https://docs.npmjs.com/misc/config#config-settings

engine-strict=true
```

Reply:
I'm not sure if npmrc is really needed for CI.

Please let's focus on the removal of Node 12 and addition of Node 20.

Reply:
Should I open a new issue for the `.npmrc` then?

Reply:
I think we are fine with just having the engines entry in package.json which throws a warning.